app/Models/Bank.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Model;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Bank extends Model
+{
+    //
+    use SoftDeletes;
+
+    protected $table = 'bank_accounts';
+
+    protected $fillable = [
+        'user_id',
+        'card_id',
+        'name',
+        'account_holder',
+        'card_number',
+        'expired_date',
+    ];
+
+    protected $table $card()
+    {
+        return $this->belongsTo('App\Models\Card', 'card_id', 'id');
```

Comment:
Card::class, cái này anh comment rồi ấy. fix hết toàn bộ

Reply:
dạ ok anh